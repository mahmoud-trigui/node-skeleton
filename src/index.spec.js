import { stringToSortedArray } from '.';

describe('example', () => {
  it('stringToSortedArray', () => {
    const result = stringToSortedArray(
      'A, list, of,    words  , without,meaning',
    );

    expect(result).toMatchSnapshot();
  });
});
