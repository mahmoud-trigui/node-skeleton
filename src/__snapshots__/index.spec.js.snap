// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example stringToSortedArray 1`] = `
Array [
  "A",
  "list",
  "meaning",
  "of",
  "without",
  "words",
]
`;
